adapter-rest-contract/src/main/resources/io/github/fileanalysissuite/adapters/rest/contract/swagger.yaml

```diff
@@ -94,12 +133,12 @@ components:
     FailureDetails:
       type: object
       properties:
-        exceptions:
```

Comment:
Is it right that the exceptions are gone altogether?

Reply:
Yes, I think messages can sort this out. They are kept in the .net adapter because it makes it easier to log, but at the end, it's turned into strings.
I don't think it makes sense to keep them here - but I'm open for discussion. We could for example have a type in the swagger.